authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,12 @@
+
+from django.urls import path
+
+
+# local imports
+from .views import GetUserProfile, UpdateUserProfile
```

Comment:
Can you change the name of the views to add View to the end, to conform to the other view?

Reply:
Kindly elaborate on this.

Reply:
I think what @bettblake08 means is that you use something like 'GetUserProfileView' or 'UpdateUserProfileView'